pkgs/top-level/all-packages.nix

```diff
@@ -23714,7 +23699,7 @@ in
   multimc = libsForQt5.callPackage ../games/multimc { };
 
   inherit (callPackages ../games/minetest {
-	inherit (darwin) libiconv;
+    inherit (darwin) libiconv;
```

Comment:
Unrelated change, my `emacs` replaced the tab character someone inserted into this file with the correct number of spaces.